### ISSUE TITLE
Search Param Table can't be altered in 4.diff.sql

### ISSUE THREAD

**Issue by rotodd** [microsoft/fhir-server#1206]
**Describe the bug**

On startup, an application startup exception is thrown:

```
Application startup exception: Microsoft.SqlServer.Management.Common.ExecutionFailureException: An exception occurred while executing a Transact-SQL statement or batch.
 ---> System.Data.SqlClient.SqlException (0x80131904): ALTER TABLE only allows columns to be added that can contain nulls, or have a DEFAULT definition specified, or the column being added is an identity or timestamp column, or alternatively if none of the prev
ious conditions are satisfied the table must be empty to allow addition of this column. Column 'Status' cannot be added to non-empty table 'SearchParam' because it does not satisfy these conditions.
   at Microsoft.SqlServer.Management.Common.ConnectionManager.ExecuteTSql(ExecuteTSqlAction action, Object execObject, DataSet fillDataSet, Boolean catchException)
   at Microsoft.SqlServer.Management.Common.ServerConnection.ExecuteNonQuery(String sqlCommand, ExecutionTypes executionType, Boolean retry)

```

The following SQL command will not work in `4.diff.sql` since non nullable columns can't be added to a table with existing data.

```
ALTER TABLE dbo.SearchParam
ADD
    Status varchar(10) NOT NULL,
    LastUpdated datetimeoffset(7) NOT NULL,
    IsPartiallySupported bit NOT NULL

```

Dev Ops bug: [#AB75439](https://microsofthealth.visualstudio.com/Health/_workitems/edit/75439)

**Comment by CaitlinV39** [comment]
@rotodd was this resolved by #1219 or is there still open work on this one?

**Comment by rotodd** [comment]
> @rotodd was this resolved by #1219 or is there still open work on this one?

@CaitlinV39 this was resolved in #1219. Sorry, I didn't realize the bug was still open! I will mark it as closed
